test/FluentMigrator.Tests/Unit/Generators/Postgres/PostgresColumnTests.cs

```diff
@@ -264,5 +264,41 @@ public override void CanRenameColumnWithDefaultSchema()
             var result = Generator.Generate(expression);
             result.ShouldBe("ALTER TABLE \"public\".\"TestTable1\" RENAME COLUMN \"TestColumn1\" TO \"TestColumn2\";");
         }
+
+        [Test]
+        public override void CanCreateColumnWithComputedExpression()
+        {
+            var expression = GeneratorTestHelper.GetCreateColumnExpressionWithComputed();
+            
+            var result = Generator.Generate(expression);
+            result.ShouldBe("ALTER TABLE \"public\".\"TestTable1\" ADD \"TestColumn1\" GENERATED ALWAYS AS (Price * Quantity) STORED NOT NULL;");
+        }
+
+        [Test]
+        public override void CanCreateColumnWithStoredComputedExpression()
+        {
+            var expression = GeneratorTestHelper.GetCreateColumnExpressionWithStoredComputed();
+            
+            var result = Generator.Generate(expression);
+            result.ShouldBe("ALTER TABLE \"public\".\"TestTable1\" ADD \"TestColumn1\" GENERATED ALWAYS AS (Price * Quantity) STORED NOT NULL;");
+        }
+
+        [Test]
+        public override void CanAlterColumnToAddComputedExpression()
+        {
+            var expression = GeneratorTestHelper.GetAlterColumnExpressionWithComputed();
+            
+            var result = Generator.Generate(expression);
+            result.ShouldBe("ALTER TABLE \"public\".\"TestTable1\" ALTER \"TestColumn1\" TYPE ;");
```

Comment:
todo

Reply:
@Shane32 @jzabroski While writing integration tests, I'm telling myself I'm not sure we need to support this, or only the possibility to change the expression of an already computed column. What do you think ?

Reply:
I think line 292 is wrong. 
1. All computed columns in Postgres are STORED.
2. You cannot change the type of the computed column.
3. You cannot replace a non-computed column with a computed column expression.

We could offer CompatibilityMode.Strict as a way to throw an error if someone tries to do this. If in non-Strict mode, drop the existing column and re-create it.